### ISSUE TITLE
Can't install sqlite3

### ISSUE THREAD

**Issue by hjrobinson** [apollographql/fullstack-tutorial#198]
I apologize if this issue has already been addressed but I can't install sqlite3. So I've tried to fix this on my own to no avail by changing node versions and installing different versions of sqlite3. Any advice is appreciated. I have Visual Studio code installed and I use Windows 8. I'm not sure if that matters.

**Comment by abiola-ajibola** [comment]
Update it to version 5.0.0

See https://github.com/mapbox/node-sqlite3/issues/1430

**Comment by hjrobinson** [comment]
Yes you are correct I needed to use this version.  By the way I erred in my statement above. I don't know if having VSCode installed matters. I also because I am using Windows 8 I needed the appropriate sdk which is not available with the latest VSCommunity. This literally took me 12 hours to figure out. So here's what I did.

1.  Fresh install of node making sure you check the box for having Chocolatey, Python, and all the extra stuff installed.

2.  Download VSCommunity 2015 from here:
https://my.visualstudio.com/Downloads/Featured?mkt=en-us

3. Then install sqlite3@5.0.0 per your advice above (either update the package.json or just run the command):
npm install sqlite3@5.0.0

So if anyone else is living in the past with Windows 8 this is how you complete the install. Good luck!

**Comment by OctavioBR** [comment]
Since the pre-built binaries are unavailable (403) from https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v5.0.1/napi-v6-linux-x64.tar.gz, `node-pre-gyp` will actually **require python** to be installed on your machine to successfully install sqlite

**Comment by abiola-ajibola** [comment]
Oh, I see, I do not have Python installed on my machine, so I used another version of SQLite.

**Comment by hjrobinson** [comment]
I just wanted to add here that I was installing the packages for the final version of the server and even after updating the package.json to install 5.0.0 it automatically installed sqlite3 version 5.0.1 . When I ran npm install sqlite3@5.0.0 sqlite3 installed correctly.

**Comment by abiola-ajibola** [comment]
> I just wanted to add here that I was installing the packages for the final version of the server and even after updating the package.json to install 5.0.0 it automatically installed sqlite3 version 5.0.1 resulting in an installation error.  When I ran npm install sqlite3@5.0.0 sqlite3 installed correctly.

You can use *5.0.0 interest of ^5.0.0 or ~5.0.0 in your package.json

**Comment by hjrobinson** [comment]
I know these things are kind of obvious but if you have a package-lock.json leftover from a failed install you will want to delete that before making your next attempt and it may be a good idea to run your installs as admin in the Windows terminal.
